src/main/surfrdUtilsMod.F90

```diff
@@ -181,10 +181,8 @@ subroutine collapse_crop_types(wt_cft, fert_cft, cftsize, begg, endg, verbose, s
     character(len=*), parameter :: subname = 'collapse_crop_types'
     !-----------------------------------------------------------------------
 
-    if (cftsize <= 0) then  ! Currently this applies only if use_fates
-       call endrun(msg = subname//' can only be called if cftsize > 0' // &
-            errMsg(sourcefile, __LINE__))
-    end if
+    if (cftsize > 0) then  ! The opposite applies only if use_fates
```

Comment:
So if use_fates then do nothing instead of aborting.

Reply:
OK